src/lsm/manifest_level.zig

```diff
@@ -701,7 +716,7 @@ pub fn TestContext(
 
                 if (to_remove.items.len > 0) {
                     for (to_remove.items) |*table| {
-                        context.level.remove_table(
+                        context.level.remove_table_invisible(
```

Comment:
Since the function signature is the same, have you double-checked that we've updated all necessary call sites in the code base to use `remove_table_invisible` instead?

Reply:
I believe so. CI caught some here in the tests previously. It helps that one version accepts snapshots while another doesn't.